src/chrome/content/rules/Shoppingshadow.com.xml

```diff
@@ -1,18 +1,21 @@
 <!--
-	For other eBay coverage, see EBay.xml.
-
-
-	Fully covered domains:
+	img and di* have certificate mismatch issue, need to redirect to secure-*
```

Comment:
Please also add test urls/rules for the `secure-` variants.

Why did you removed the reference to `EBay.xml`?